airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -113,8 +122,14 @@ def execute(self, context: Context) -> list[str]:
         steps = self.steps
         if isinstance(steps, str):
             steps = ast.literal_eval(steps)
-
-        return emr_hook.add_job_flow_steps(job_flow_id=job_flow_id, steps=steps, wait_for_completion=True)
+        # from pdb import set_trace
+        # set_trace()
```

Comment:
Debugging code you forgot to delete?

(Also fyi you can use `breakpoint()` for the same effect without importing pdb)

Reply:
Oops, that's embarrassing 😅. But I didn't know about the `breakpoint()` method, so it looks like I learned something 😃